assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -606,11 +608,17 @@ che.infra.kubernetes.runtimes_consistency_check_period_min=-1
 # OpenShift infrastructure reuse most of the Kubernetes configuration attributes.
 #
 
+# DEPRECATED - please do not change the value of this property otherwise the existing workspaces will loose data. Do not
+# set it on new installations.
+#
```

Comment:
As above, I don't think a user of Che who is upgrading to 7.3.0 or higher will see this warning.